chat_shell/chat_shell/tools/builtin/create_subscription.py

```
@@ -0,0 +1,557 @@
+        trigger_type: Literal["cron", "interval", "one_time"],
+        prompt_template: str,
+        description: Optional[str] = None,
+        cron_expression: Optional[str] = None,
+        interval_value: Optional[int] = None,
+        interval_unit: Optional[Literal["minutes", "hours", "days"]] = None,
+        execute_at: Optional[str] = None,
+        preserve_history: bool = False,
+        history_message_count: int = 10,
+        retry_count: int = 1,
+        timeout_seconds: int = 600,
+        run_manager: CallbackManagerForToolRun | None = None,
```

Ruff (0.14.13)

Unused method argument: `run_manager`

(ARG002)